ui/src/data_explorer/containers/DataExplorer.tsx

```diff
@@ -265,6 +298,8 @@ const mapDispatchToProps = dispatch => {
       dataExplorerActionCreators,
       dispatch
     ),
+    handleGetDashboards: bindActionCreators(getDashboardsAsync, dispatch),
```

Comment:
(nitpick) this isn't consistent across the codebase (not even used here) but I've been digging the following naming convention lately:

- callbacks passed as props are named like `onFoo`
- callbacks defined and used within the same class are named like `handleFoo`

so in this case it would be `onHandleGetDashboards` and `onAddDashboardCell`